index.js

```diff
@@ -0,0 +1,68 @@
+let addMessage = document.querySelector(".myInput"),
```

Comment:
Its better to use id based selectors since there can be many elements with `myInput` classname. its not safe for big projects